_includes/v21.1/spatial/zmcoords.md

```diff
@@ -0,0 +1,24 @@
+<span class="version-tag">New in v21.1:</span> You can also store a `{{page.title}}` with the following additional dimensions:
+
+- A third dimension coordinate `Z` (`{{page.title}}Z`).
+- A measure coordinate `M` (`{{page.title}}M`).
+- Both a third dimension and a measure coordinate (`{{page.title}}ZM`).
+
+The `Z` and `M` dimensions are accessible using a number of [built-in functions](functions-and-operators.html#spatial-functions), including:
```

Comment:
I would replace "accessible" with "Z and M dimensions can be accessed or modified using a number of ..."

Reply:
Updated!